src/lib/Graph.coffee

```diff
@@ -33,7 +34,7 @@ class Graph extends EventEmitter
   # and giving it a name:
   #
   #     myGraph = new Graph 'My very cool graph'
-  constructor: (@name = '') ->
+  constructor: (@name = '', @caseSensitive = false) ->
```

Comment:
@ifitzpatrick I think name can go to options if you ensure backwards compat with something like:

```coffee
constructor: (options) ->
  if typeof options is 'string'
    options =
      name: options
```